src/Common/ErrorCodes.cpp

```diff
@@ -546,6 +546,7 @@
     M(577, INVALID_SHARD_ID) \
     M(578, INVALID_FORMAT_INSERT_QUERY_WITH_DATA) \
     M(579, INCORRECT_PART_TYPE) \
+    M(580, UNKNOWN_HANDLE_KAFKA_ERROR_MODE) \
```

Comment:
Maybe just use `ErrorCodes::BAD_ARGUMENTS` ?

Reply:
It's better to use bad arguments. I'll fix it.